### PR TITLE
Periodically run test to verify IAP is working on auto-deployed clusters

### PR DESCRIPTION
**Pull request by jlewi**
* #51: We want to periodically run a test to verify that IAP is
  working on the auto-deployed clusters.

* Document the testing.

### CODE REVIEW

**Comment by jlewi** [comment]
Test failure was because docker image in the tekton task was outdated.
/test all

File: prow_config.yaml [hunk 1]
```diff
@@ -3,3 +3,14 @@
 workflows:
```
Comment (Bobgy) [inline prow_config.yaml:3]: I guess my previous question was, how can I use `prow_config.json` in pipelines repo?

File: README.md [hunk 2]
```diff
@@ -17,4 +17,12 @@ For more information about blueprints refer to the [kpt blueprint guide](https:/
 
 1. Use the [management](./management/README.md) blueprint to spin up a management
    cluster
-1. Use the [kubeflow](./kubeflow/README.md) blueprint to create a Kubeflow deployment.
\ No newline at end of file
+1. Use the [kubeflow](./kubeflow/README.md) blueprint to create a Kubeflow deployment.
+
+## Development
+
+### Test Grid
```
Comment (Bobgy) [inline README.md:24]: Thanks! These readmes are really helpful for finding all the pieces out there.

**Comment by rmgogogo** [comment]
/lgtm

thanks

**Pull request merged**